addons/website/views/snippets/snippets.xml

```diff
@@ -153,6 +153,7 @@
                 <t t-snippet="website.s_company_team" string="Team" group="people">
                     <keywords>organization, structure</keywords>
                 </t>
+                <t t-snippet="website.s_company_team_basic" string="Team" group="people"/>
```

Comment:
No keywords ? 
I added the `needs-keywords` tag in the task so Lionel will provide them in a follow-up task 😉
Could you check the other snippets you're handling and eventually add the tag there as well ?